src/less/components/brukernotifikasjoner/Beskjed.less

```diff
@@ -4,7 +4,7 @@
   display: flex !important;
```

Comment:
Kanskje vi etter hvert kan se på om det går an å bli kvitt !important her? :)

Reply:
Jepp, det ligger i todo listen min :). Tror det kan løses med css specificity, slik det opprinnelig var gjort for Infomeldinger.